SecurityManager/Core/Web.config

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
better name, pelase. `web.config` would be misleading, `app.config` isn't helpful, how about `dbschema.config`?